### PR TITLE
Definitions for prop-types and create-react-class

### PR DESCRIPTION
**Pull request by acdlite**
I'm doing something wrong with the definition for `create-react-class`. The test file is not erroring where it should.

@samwgoldman

### CODE REVIEW

File: definitions/npm/create-react-class_v15.x.x/flow_v0.43.x/create-react-class_v15.x.x.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+declare module 'create-react-class' {
+  declare module.exports: React$CreateClass;
```
Comment (samwgoldman) [inline definitions/npm/create-react-class_v15.x.x/flow_v0.43.x/create-react-class_v15.x.x.js]: I think you want `declare export default` here. `declare var exports` also works.

Comment (acdlite) [inline definitions/npm/create-react-class_v15.x.x/flow_v0.43.x/create-react-class_v15.x.x.js]: `declare export default` didn't work but `declare var exports` did. Thanks!

**Comment by raitucarp** [comment]
What is the status of this PR?

**Comment by bvaughn** [comment]
Ping @acdlite

**Comment by samwgoldman** [comment]
I think I can push to this PR... let me see.

**Comment by acdlite** [comment]
Thanks @samwgoldman! Sorry I missed your comment from last week

**Comment by samwgoldman** [comment]
Not your fault. I dragged my feet on review. Sure you've already moved on to bigger things :)

**Pull request merged**